test/packages/custom_logs/fields/input.yml

```diff
@@ -0,0 +1,4 @@
+- name: input.name
```

Comment:
I think this field was called `input.type` in Beats. Are we intentionally using `input.name` now?

Reply:
No no, it's just a sample, nothing mandatory. It's here to show that you can define input fields.